specific/cmo_project/models/project.py

```diff
@@ -293,15 +315,116 @@ def _get_state_before_inactive(self):
     @api.multi
     def _set_project_analytic_account(self):
         for project in self:
-            parent_project = self.env['project.project'].browse(project.project_parent_id.id)
+            parent_project = self.env['project.project'].browse(
+                project.project_parent_id.id)
             project.parent_id = parent_project.analytic_account_id.id
 
     @api.multi
     @api.constrains('brief_date', 'date')
     def _check_brief_dates(self):
         self.ensure_one()
         if self.brief_date > self.date:
-            return ValidationError("project brief-date must be lower than project end-date.")
+            return ValidationError("project brief-date must be lower than \
+                project end-date.")
+
+    @api.multi
+    def quotation_relate_project_tree_view(self):
```

Comment:
For a button click that redirect to another action window. If the action already available, you can use,
```
    @api.multi
    def open_some_action_window(self):
        self.ensure_one()
        action = self.env.ref('account_budget_activity.'
                              'act_account_budget_view')
        result = action.read()[0]
        dom = ['|', ('ref_budget_id', '=', self.id), ('id', '=', self.id)]
        result.update({'domain': dom})
        return result
```